crates/loam-cli/src/commands/build/clients.rs

```diff
@@ -62,8 +69,8 @@ impl Args {
         Ok(())
     }
 
-    fn loam_env(self) -> String {
-        format!("{:?}", self.env).to_lowercase()
+    fn loam_env(self, default: LoamEnv) -> String {
+        self.env.unwrap_or(default).to_string()
```

Comment:
```suggestion
        self.env.unwrap_or(default).to_lowercase()
```
It returns a `String` and we still want to make sure it's lowercase.

Reply:
good catch